src/back/services/update.ts

```diff
@@ -163,9 +177,9 @@ export class UpdateService {
     const update = await UpdateModel.createUpdate(data)
     try {
       if (financial_records) await FinancialService.createRecords(update.id, financial_records)
-      await DiscourseService.createUpdate(update, proposal.title)
-      await EventsService.updateCreated(update.id, proposal.id, proposal.title, user)
-      DiscordService.newUpdate(proposal.id, proposal.title, update.id, user)
+      await DiscourseService.createUpdate(update, title)
+      await EventsService.updateCreated(update.id, proposal_id, title, user)
```

Comment:
This makes me think that if updates are created for projects, then there might be some event copies that we'll need to change